src/helics/shared_api_library/api-data.h

```diff
@@ -45,8 +45,8 @@ extern "C"
     const helics_time helics_time_zero = 0.0; /*!< definition of time zero-the beginning of simulation*/
     const helics_time helics_time_epsilon = 1.0e-9; /*!< definition of the minimum time resolution*/
     const helics_time helics_time_invalid = -1.785e39; /*!< definition of an invalid time that has no meaning*/
```

Comment:
Is any adjustment to the helics invalid time also needed?

Reply:
I will look into that

Reply:
I don't believe so, there is no equivalent to that in the C++ API since it would use exceptions where the invalid time would have been used.